README.md

```diff
@@ -1,17 +1,36 @@
 # MinSQL
 
-Massively parallel log search engine with a simple HTTP API. For usage instructions, click [here](#Usage).
+Massively Parallel Processing (MPP) log database with a simple HTTP API. For usage instructions, click [here](#Usage).
 
-MinSQL ingests and searches petabytes of data distributed across MinIO buckets. MinSQL relies on the high speed data access capabilities of MinIO Object Storage and effieciency of the parquet data format to achieve massive scale.
+MinSQL stores data in Columnar-Parquet format ordered by time. It is built on top of object storage for persistance. Logs can grow indefinitely across multiple MinIO clusters.
 
-Each data unit in MinSQL is a JSON document. There are no restrictions on the schema of these documents.
+## What is a log database
+
+Log database is a type of database that is optimized for ingesting JSON data records (log lines, events, messages etc.) at massive scales with SQL-like search/query capabilities. 
 
 ## Features
 
-- [x] Petascale
-- [x] Ingest
-- [x] Search
-- [x] Shared Nothing
+- [x] Ingestion
+- [x] Search by applying operators on key/value
+- [x] Shared Nothing Architecture
+- [x] Parquet data storage format
+- [x] Petabyte scale
+- [x] Backed by MinIO
+- [x] S3 Select compatibility
+- [x] Search across MinIO clusters and buckets
+- [x] Simple HTTP Interface
+- [x] JSON support
```

Comment:
Is JSON support a feature?, I guess JSON schema evolution is a better support to show.